src/scss/packages/usa-image-example/src/_styles.scss

```diff
@@ -0,0 +1,41 @@
+@use 'uswds-core' as *;
+
+@mixin usa-image-example($color) {
+  img {
```

Comment:
To the previous comment, should this be included in the `.usa-image-example__image` styles?

Reply:
Ah, missed those. I'll switch all of the mixins to use the correct classes.

Reply:
Done ([commit](https://github.com/18F/identity-design-system/pull/369/commits/db1511fe354aae3f53c25f15ef855cf2ebf52d56)).